stamp.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -e # Exit on errors
+# set -x # Shell debugging
+
+# First, check that we're on the main branch.
+BRANCH=$(git rev-parse --abbrev-rev HEAD)
+if [ "$BRANCH" != "main" ] ; then 
+    echo "Not on main, exiting!" 
+    exit 1
+fi 
+
+# And check that the main branch is clean
+STATUS=$(git status --porcelain)
+if [ -n "${STATUS}" ] ; then
+    echo "Repository is not clean, exiting!" 
+    exit 1
+fi 
+
+# And check that the REMOTE points to the -tools repo.
+REMOTE="${REMOTE:origin}"
```

Comment:
This line ends up with `REMOTE=''` on git bash (on Windows).

Reply:
Done